proposals/umoci.md

```diff
@@ -0,0 +1,113 @@
+
+## Proposal ##
+Change the ownership of the existing umoci project from openSUSE:
+
+  https://github.com/openSUSE/umoci
+
+And move it inside the `opencontainers` organisation:
+
+  https://github.com/opencontainers/umoci
+
+The import paths will correspondingly be "github.com/opencontainers/umoci" (umoci does have some Go API users, but since the project will be renamed -- and GitHub will add a redirect -- there will be no significant downstream impact of the change).
+In future we may opt to use vanity imports (such as "umo.ci/cmd/umoci").
+
+The project's domain "umo.ci" will also be transferred to the Linux Foundation so that it can be managed by someone other than the maintainers (though maintainers must maintain the necessary administrative access to maintain the website).
+
+### Initial Maintainers ###
+Initial maintainers of the umoci project would be:
+
+* Aleksa Sarai <cyphar@cyphar.com> (@cyphar)
+* Tycho Andersen <tycho@tycho.ws> (@tych0)
+* Vincent Batts <vbatts@hashbangbash.com> (@vbatts)
+
+### Code of Conduct ###
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md
+
+### Governance and Releases ###
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+It should be noted that since umoci is not a specification, it is not bound by the ordinary quorum and voting rules for specification release.
+As such, new versions will be released as regularly as needed without the need for a quorum vote.
+This is to avoid the serious delays in releases encountered within the runc project.
+
+Until there are enough additional maintainers, PRs will be merged with only one maintainer LGTM required.
+Maintainers are encouraged to not merge their own PRs immediately (preferably waiting at least 24 to 48 hours to allow another maintainer to review it), but this is not a strict rule.
+
+### Project Communications ###
+The proposed project would continue to use existing channels in use by the OCI developer community for communication including:
+
+ * GitHub for issues and pull requests.
+ * The [`dev@opencontainers.org`][oci-ml] email list.
+ * The weekly OCI developer community conference call.
+ * The `#opencontainers` IRC channel on Freenode.
+ * The [OCI Slack workspace][oci-slack].
+ * The [OCI Matrix Room][oci-matrix].
+
+[oci-ml]: mailto:dev@opencontainers.org
+[oci-slack]: https://opencontainers.slack.com/
+[oci-matrix]: https://matrix.to/#/#opencontainers:matrix.org
+
+### Roadmap ###
+umoci still has a fairly large scope of work, and this work would continue under OCI.
+Examples of such work include:
+
+ * Hookable execution to allow for custom storage drivers to be defined by users (to avoid the maintenance overhead of supporting such drivers within umoci).
```

Comment:
what type of storage do you mean here? just basic file system io, or some image caching?

Reply:
Probably both, the idea would be that you would have hookable points in `runc`'s extraction of layers such that you could skip layers (image caching) or set up mountpoints (giving you a proper overlay filesystem) or any number of other things.

Reply:
nod... important then to highlight this as a part of the proposed scope as there will likely be a lot of discussion around the subject.